### PR TITLE
Add podAnnotations and podLabels to values and docs

### LINKED ISSUE #803: `.Values.podLabels` and `.Values.podAnnotations` are missing in `values.yaml`
**What version of descheduler are you using?**

descheduler version: 0.23.1

**Does this issue reproduce with the latest release?**

Yes

**What did you do?**

Note that CronJob and Deployment helm templates refer to `.Values.podLabels` and `.Values.podAnnotations`, but those are not defined in `values.yaml`.

**What did you expect to see?**

`.Values.podLabels` and `.Values.podAnnotations` to be defined in `values.yaml`

### PR DESCRIPTION
**Pull request by stephan2012**
Closes #803

### CODE REVIEW

File: charts/descheduler/README.md [hunk 1]
```diff
@@ -69,6 +69,8 @@ The following table lists the configurable parameters of the _descheduler_ chart
 | `serviceAccount.create`             | If `true`, create a service account for the cron job                                                                  | `true`                               |
 | `serviceAccount.name`               | The name of the service account to use, if not set and create is true a name is generated using the fullname template | `nil`                                |
 | `serviceAccount.annotations`        | Specifies custom annotations for the serviceAccount                                                                   | `{}`                                 |
+| `podAnnotations`                    | Annotations to add to Pods                                                                                            | `{}`                                 |
+| `podLabels`                         | Labels to add to Pods                                                                                                 | `{}`                                 |
```
Comment (damemi) [inline charts/descheduler/README.md:73]: maybe clarify that this is added to the descheduler Pods?

**Review by damemi — COMMENTED** [review]
/ok-to-test
/approve

File: charts/descheduler/README.md (hunk 1 above)
Comment (stephan2012) [reply charts/descheduler/README.md]: Done.

**Pull request merged**